### PR TITLE
Pin 1330 authorization-process GET /clients/{clientId}/keys/{kid}

### PR DESCRIPTION
**Pull request by nttdata-rtorsoli**

### CODE REVIEW

File: src/main/resources/interface-specification.yml [hunk 1]
```diff
@@ -6751,6 +6751,103 @@ paths:
...
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: returns the corresponding key
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/PublicKey'
```
Comment (galales) [inline src/main/resources/interface-specification.yml:6780]: I think this endpoint is used to return the actual value of the public key, but `PublicKey` model does not contain this information

File: src/main/scala/it/pagopa/interop/backendforfrontend/api/impl/ClientsApiServiceImpl.scala [hunk 2]
```diff
@@ -288,6 +295,20 @@ final case class ClientsApiServiceImpl(
     }
   }
 
+  private def decorate(
```
Comment (galales) [inline src/main/scala/it/pagopa/interop/backendforfrontend/api/impl/ClientsApiServiceImpl.scala]:
```suggestion
  private def decorateKey(
```

**Review by galales — CHANGES_REQUESTED** [review]

File: src/main/resources/interface-specification.yml (hunk 1 above)
Comment (galales) [reply src/main/resources/interface-specification.yml:6780]: I probably mistook this endpoint for `/clients/{clientId}/encoded/keys/{keyId}`.
Let's wait for frontend confirmation and then on my side we can proceed

**Review by galales — APPROVED** [review]

File: src/main/resources/interface-specification.yml (hunk 1 above)
Comment (nttdata-rtorsoli) [reply src/main/resources/interface-specification.yml:6780]: Further investigations are required in a next refinement

**Review by beetlecrunch — APPROVED** [review]

**Pull request merged**